### ISSUE TITLE
Error on console.

### ISSUE THREAD

**Issue by kodeine** [MurhafSousli/ng2-sharebuttons#8]
Hello,
I'm getting following error on console.

```html
                    <share-buttons
                            [url]="requestLink"
                            [count]="true"
                            [totalCount]="false"

                            [pinterest]="false"
                            [linkedIn]="false"
                            [tumblr]="false"
                            [reddit]="false"
                            [stumbleUpOn]="false"
                    ></share-buttons>
```

![screenshot from 2016-10-20 21-39-21](https://cloud.githubusercontent.com/assets/8620059/19584933/d7ff4a56-970d-11e6-8f9a-e662afa2c052.png)
![screenshot from 2016-10-20 21-39-40](https://cloud.githubusercontent.com/assets/8620059/19584934/da5bb528-970d-11e6-9d58-9900d9ea91df.png)

**Comment by MurhafSousli** [comment]
Hi @kodeine, apparently this is because the link didn't get count from fb, I will fix this very soon and let you know when updates are pushed

**Comment by MurhafSousli** [comment]
@kodeine fixed in 1.0.6
